aait/web/group-1/app/blog/[blogId]/page.tsx

```diff
@@ -0,0 +1,94 @@
+"use client";
+
+import { useGetSingleBlogQuery } from "@/store/features/blogs/blogs";
+import { Blog } from "@/types/Blog";
+import Image from "next/image";
+import { useParams } from "next/navigation";
+import React from "react";
+
+const page = () => {
+  const { blogId } = useParams();
+  const {
+    data: blog,
+    isLoading,
+    isError,
+    error,
+  } = useGetSingleBlogQuery(blogId);
+
+  return (
+    <div>
+      {isLoading && (
+        <div className="text-center font-bold font-montserrat text-xl">
+          Loading...
+        </div>
+      )}
+      {isError && (
+        <div className="text-center font-bold font-montserrat text-xl">
+          {(error as any).data
+            ? (error as any).data?.error
+            : "An Error Occured while loading the data"}
+        </div>
+      )}
+      {!isLoading && !isError && (
```

Comment:
Use `return `  in each code of block instead of using `!isLoading && !isError &&`